app/services/os-datastore.js

```diff
@@ -0,0 +1,415 @@
+'use strict';
+
+var _ = require('underscore');
+var crypto = require('crypto');
+var Promise = require('bluebird');
+require('isomorphic-fetch');
+
+var defaultOptions = {
+  conductorUrl: 'http://os-conductor.herokuapp.com/datastore/',
```

Comment:
Actually, it is used. But since it is needed on frontend, it is passed to Angular app through controller